iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/ExcessivePermissionsCheck.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.terraform.checks.gcp;
+
+import java.util.List;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.sonar.check.Rule;
+import org.sonar.check.RuleProperty;
+import org.sonar.iac.common.api.checks.SecondaryLocation;
+import org.sonar.iac.common.checks.TextUtils;
+import org.sonar.iac.terraform.api.tree.ExpressionTree;
+import org.sonar.iac.terraform.checks.AbstractNewResourceCheck;
+import org.sonar.iac.terraform.symbols.ListSymbol;
+
+@Rule(key = "S6406")
+public class ExcessivePermissionsCheck extends AbstractNewResourceCheck {
+
+  private static final String MESSAGE = "This role grants more than %d sensitive permissions. Make sure they are all required.";
+  private static final String SECONDARY_MESSAGE = "Sensitive permission";
+  public static final int DEFAULT = 5;
+
+  private static final Set<String> SENSITIVE_FEATURE_SUFFIXES = Set.of("abort", "access", "add", "allocate", "analyze", "apply", "approve", "associate", "attach", "begin",
```

Comment:
Couple of quotes from the ticket:
_We will refer to them as PREFIX, FEATURE, and SUFFIX, to avoid ambiguities._
_The SUFFIX starts with one of these words, OR is equal to one of these words ..._

Thus I suggest renaming this to **SENSITIVE_SUFFIX_PREFIXES** or **SENSITIVE_FEATURE_SUFFIX_PREFIXES**

Reply:
WDYT about SENSITIVE_ACTION?